docs/data/joy/components/stack/InteractiveStack.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import FormControl from '@mui/joy/FormControl';
+import FormLabel from '@mui/joy/FormLabel';
+import Grid from '@mui/joy/Grid';
+import BrandingProvider from 'docs/src/BrandingProvider';
+import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
+import Sheet from '@mui/joy/Sheet';
+import RadioGroup from '@mui/joy/RadioGroup';
+import Radio from '@mui/joy/Radio';
+import Stack, { StackProps } from '@mui/joy/Stack';
+
+export default function InteractiveStack() {
+  const [direction, setDirection] = React.useState<StackProps['direction']>('row');
+  const [justifyContent, setJustifyContent] = React.useState('center');
+  const [alignItems, setAlignItems] = React.useState('center');
+  const [spacing, setSpacing] = React.useState(2);
+
+  const jsx = `
+<Stack
+  direction="${direction}"
+  justifyContent="${justifyContent}"
+  alignItems="${alignItems}"
+  spacing={${spacing}}
+>
+`;
+
+  return (
+    <Stack sx={{ flexGrow: 1 }}>
+      <Stack
+        direction={direction}
+        justifyContent={justifyContent}
+        alignItems={alignItems}
+        spacing={spacing}
+        sx={{ height: 240 }}
+      >
+        {[0, 1, 2].map((value) => (
+          <Sheet
+            key={value}
+            sx={{
+              p: 2,
+              pt: value + 1,
+              pb: value + 1,
+              typography: 'body2',
+            }}
+          >
+            {`Item ${value + 1}`}
+          </Sheet>
+        ))}
+      </Stack>
+      <Sheet sx={{ p: 2 }}>
+        <Grid container spacing={3}>
+          <Grid xs={12}>
+            <FormControl>
+              <FormLabel sx={{ mb: 1.5 }}>direction</FormLabel>
+              <RadioGroup
+                row
+                name="direction"
+                aria-label="direction"
+                value={direction}
+                onChange={(event) => {
+                  setDirection(event.target.value as StackProps['direction']);
+                }}
```

Comment:
It does not look good.
<img width="771" alt="image" src="https://user-images.githubusercontent.com/18292247/206657408-162aabaf-4a8b-4c6a-923e-ad8f34e3401e.png">

```suggestion
                }}
                sx={{ flexWrap: 'wrap', gap: 2, '--RadioGroup-gap': '0px' }}
```